stable/database/templates/statefulset.yaml

```diff
@@ -352,6 +364,13 @@ spec:
         resources:
 {{ toYaml .Values.database.sm.resources | trim | indent 10 }}
         volumeMounts:
+        {{- if .Values.database.configFiles }}
+        {{- range $key, $val := .Values.database.configFiles }}
```

Comment:
And one last one - just it doesn't get overlooked...

Reply:
Fixed